pubsub/api/PubsubSample/Program.cs

```diff
@@ -195,7 +195,7 @@ public class Pubsub
         public static object CreateTopic(string projectId, string topicId)
         {
             // [START create_publisher_client]
-            PublisherClient publisher = PublisherClient.Create();
+            PublisherServiceApiClient publisher = PublisherServiceApiClient.Create();

```

Comment:
minor: It would be easier if the cases when the type of the variable is clear from the assignment, to use "var".
For example,

var publisher = PublisherServiceApiClient.Create();

That way refactoring are easier, you write less code without sacrificing the readability.

Reply:
True, but having explicit types (not `var`) can be useful in samples, as it makes the type clearer for users who are not familiar with our code.
And this code shouldn't be changed again. We expect this to be the final beta release before the GA.